unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -61,7 +71,14 @@ CreatorLocks.defaultProps = {
   showForm: false,
 }
 
-export default CreatorLocks
+const mapDispatchToProps = dispatch => ({
+  createLock: lock => dispatch(createLock(lock)),
+})
+
+export default connect(
+  undefined,
```

Comment:
Do you mind adding a quick comment after undefined to show that there is no `mapStateToProps` ? I worry that future us will be looking for what that `undefined` is.

Reply:
sure, good idea